content/_changelogs/10.11.0.md

```diff
@@ -0,0 +1,32 @@
+## 10.11.0
+
+_Released 10/25/2022_
+
+**Features:**
+
+- Adds support for [`cy.session()`](/api/commands/session) to experimental WebKit.
```

Comment:
Can we link to the webkit experiment.

Also nit: I initially read this as "added support for cy.session()" which me since it's been around awhile. Maybe

Reply:
@emilyrohrbough I updated the wording of this issue. Let me know if you think it still needs adjusting.